lib/ex_unit/test/ex_unit/doc_test_test.exs

```diff
@@ -640,8 +640,9 @@ defmodule ExUnit.DocTestTest do
   end
 
   test "fails in indentation mismatch" do
-    message =
-      ~r[test/ex_unit/doc_test_test\.exs:\d+: indentation level mismatch: "   iex> bar = 2", should have been 2 spaces]
```

Comment:
I would keep the regex:

`~r[test/ex_unit/doc_test_test\.exs:\d+: indentation level mismatch on doctest line: "   iex> bar = 2".*is exactly 2 spaces]s`

Reply:
Inclined to agree. I’ll be at a computer in an hour, then I’ll tidy that up.

Reply:
Yeah, let's keep the regexes so we don't have to change tests on every small change. :)